File: app/src/main/java/com/inspiringteam/xchange/util/ChromeTabsUtils/ServiceConnectionCallback.java
package com.inspiringteam.xchange.util.ChromeTabsUtils;

import androidx.browser.customtabs.CustomTabsClient;


public interface ServiceConnectionCallback {
    /**
     * Called when the service is connected.
     *
     * @param client a CustomTabsClient
     */
    void onServiceConnected(CustomTabsClient client);

    /**
     * Called when the service is disconnected.
     */
    void onServiceDisconnected();
}

File: app/src/main/java/com/inspiringteam/xchange/util/chromeTabsUtils/ServiceConnectionCallback.kt
package com.inspiringteam.xchange.util.chromeTabsUtils

import androidx.browser.customtabs.CustomTabsClient

interface ServiceConnectionCallback {
    /**
     * Called when the service is connected.
     *
     * @param client a CustomTabsClient
     */
    fun onServiceConnected(client: CustomTabsClient)

    /**
     * Called when the service is disconnected.
     */
    fun onServiceDisconnected()
}